lib/tasks/local_census_records.rake

```diff
@@ -0,0 +1,11 @@
+namespace :local_census_records do
+  desc "Remove duplicated records from database"
+  task remove_duplicates: :environment do
+    ApplicationLogger.new.info "Removing local census records duplicates"
+    ids = LocalCensusRecord.group(:document_type, :document_number).pluck("MIN(id) as id")
+
+    duplicates = LocalCensusRecord.count - ids.size
+    LocalCensusRecord.where("id NOT IN (?)", ids).destroy_all
+    ApplicationLogger.new.info "Removed #{duplicates} records."
```

Comment:
I'm thinking the log messages might generate unneeded noise during deployments :thinking:. I mean, now this task will be executed every time you deploy between the moment you upgrade to version 1.1 and the moment you upgrade to version 1.2, but duplicates will only be deleted once.

So what about something like:

```ruby
    ids = LocalCensusRecord.group(:document_type, :document_number).pluck("MIN(id) as id")
    duplicates = LocalCensusRecord.count - ids.size

    if duplicates > 0
      ApplicationLogger.new.info "Removing local census records duplicates"
      LocalCensusRecord.where("id NOT IN (?)", ids).destroy_all
      ApplicationLogger.new.info "Removed #{duplicates} records."
    end
```

Another option would be to check whether the migration to add the unique index has already been run :thinking:.

Reply:
Of course, better to have the minimum noise possible!

Reply:
Done!